params/params.yaml

```diff
@@ -203,7 +215,7 @@ vesc_driver_node:
     duty_cycle_min: 0.0
     position_max: 0.0
     position_min: 0.0
-    servo_max: 0.85
+    servo_max: 0.75
```

Comment:
I think this should be 0.7 to have the steering centered around the `steering_angle_to_servo_offset`